packages/core/src/routes/dashboard.ts

```diff
@@ -1,12 +1,57 @@
+import dayjs, { Dayjs } from 'dayjs';
+
+import { getDailyNewUserCountsByTimeInterval } from '@/queries/log';
 import { countUsers } from '@/queries/user';
 
 import { AuthedRouter } from './types';
 
+const getDateString = (day: Dayjs) => day.format('YYYY-MM-DD');
+
 export default function dashboardRoutes<T extends AuthedRouter>(router: T) {
   router.get('/dashboard/users/total', async (ctx, next) => {
     const { count: totalUserCount } = await countUsers();
     ctx.body = { totalUserCount };
 
     return next();
   });
+
+  router.get('/dashboard/users/new', async (ctx, next) => {
+    const today = dayjs();
+    const fourteenDaysAgo = today.subtract(14, 'day');
+    const dailyNewUserCounts = await getDailyNewUserCountsByTimeInterval(
```

Comment:
Should be singlar form, `Count`?

Reply:
There is a group of multiple counts. Maybe better with `*Counts`

Reply:
ok